supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/statuspolling/JesApiQueryManager.scala

```diff
@@ -27,6 +28,13 @@ class JesApiQueryManager(val qps: Int Refined Positive) extends Actor with Actor
 
   private implicit val ec = context.dispatcher
   private val maxRetries = 10
+  // FIXME: Determined empirically that we start to get errors when the batch request approaches 15MB.
+  // Find out what the real value is
+  private val maxBatchRequestSize = 14 * 1024 * 1024
+  private val requestTooLargeException = new JesApiException(new IllegalArgumentException(
+    "The JES creation request exceeds the maximum size allowed. " +
+      "If you have a task with a very large number of inputs and / or outputs in your workflow you should try to reduce it."
```

Comment:
I feel like that's in the zone of "there's probably a better way to do what you're doing" but that's also likely to wind up in the "this is a fire! fix it!" zone as well. Even if the ultimate answer is "change what you're doing", keep in mind that we'll need to context switch to taking care of them when determining risk/reward of leaving it this way